src/main/java/frc/robot/subsystems/ball/intake/IntakeImpl.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems.ball.intake;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import ca.team3161.lib.robot.subsystem.RepeatingPooledSubsystem;
+import java.util.concurrent.TimeUnit;
+
+public class IntakeImpl extends RepeatingPooledSubsystem implements Intake {
+
+    Solenoid intakeSolenoid;
+    WPI_TalonSRX intakeMotorController;
+    boolean extended;
+    int intakeDirection;
+    double intakeSpeed;
+
+    public IntakeImpl() {
+        super(250, TimeUnit.MILLISECONDS); // Figure out actual value
```

Comment:
Unless the task has a specific reason for running slowly, I'd use something like 20ms as the standard value. Even if it's for a system that doesn't appear to really have a need for re-computing its state quickly, like an intake (as opposed to a PID controller for example), the resource drain overhead of running a simple task like this 50 times per second is pretty low. Minimal downsides. On the upside, a shorter period also means shorter and more consistent latency between command and action.

Reply:
Yeah sure I can do that, just wasn't sure what an appropriate value was.